editor.planx.uk/src/routes/views/flowEditor.tsx

```diff
@@ -144,3 +163,45 @@ export const flowEditorView = async (req: NaviRequest) => {
     </FlowEditorLayout>
   );
 };
+
+const includedServices = [
+  getFOIYNPP(environment),
+  getRAB(environment),
+  getSubmission(environment),
+  getDiscretionary(environment),
+];
+
+export const generateAnalyticsLink = ({
```

Comment:
I'd move these to `analytics/utils.ts` as well - they're analytics specific and aren't related to the routing and data fetching logic this file is responsible for.